barcodes_generator_product/models/product_template.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (C) 2014-Today GRAP (http://www.grap.coop)
+# Copyright (C) 2016-Today La Louve (http://www.lalouve.net)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+from odoo.addons.barcodes_generator_abstract.models.barcode_rule import _GENERATE_TYPE
```

Comment:
```suggestion
```
Same as https://github.com/OCA/stock-logistics-barcode/pull/313/commits/d7806ed90ef02e6506d66c60159e86bbc17d381a

Reply:
I think, here does not apply because it's a diferent module